### PR TITLE
Fix certain bytes were incorrectly replaced to EFBFBD

### PR DESCRIPTION
**Pull request by tanaga9**
Changed to detect text files instead of binary files. #49
Please merge if there is no problem.

### CODE REVIEW

**Comment by tanaga9** [comment]
Check Release Failed.
Is it because of my codes ?

**Comment by jtpio** [comment]
Thanks @tanaga9.



> Check Release Failed.
> Is it because of my codes ?

It's also happening on a new PR so it's not related: https://github.com/jupyterlab-contrib/jupyterlab-filesystem-access/pull/51. I'll try to fix it in https://github.com/jupyterlab-contrib/jupyterlab-filesystem-access/pull/52

File: src/drive.ts [hunk 1]
```diff
@@ -374,13 +374,13 @@ export class FileSystemDrive implements Contents.IDrive {
...
-    // We assume here image, audio and video mimetypes are all and only binary files we'll encounter
-    if (
-      file.type &&
-      file.type.split('/') &&
-      ['image', 'audio', 'video'].includes(file.type.split('/')[0])
-    ) {
-      format = 'base64';
+    // We assume here text mimetypes are all and only text files we'll encounter
+    if (file.type && file.type.split('/')) {
+      if (['text'].includes(file.type.split('/')[0])) {
```
Comment (jtpio) [inline src/drive.ts:379]: This check seems to be treating JSON files (`application/json`) differently now.

Noticed while testing this PR on https://jupyterlab-filesystem-access--50.org.readthedocs.build/en/50/lite/lab/index.html:

image

Comment (tanaga9) [reply src/drive.ts:379]: oops

**Comment by tanaga9** [comment]
Sorry, I can't remember the details anymore, so I'll close it for now.

**Pull request closed without merging**